LambaAnalyzer/mcconfigfile/crab_config.py

```diff
@@ -0,0 +1,19 @@
+from CRABClient.UserUtilities import config
+config = config()
+
+config.General.requestName = 'crab1000' #just a name of the job
+config.General.workArea = 'crab_projects'
+config.General.transferOutputs = True
+
+config.JobType.pluginName = 'PrivateMC'
+config.JobType.psetName = 'crab_MC1000.py'
+
```

Comment:
Could you change this to one of the python files (whichever one is right) that Brandon uploaded? Otherwise it's gonna get confusing. Are there any significant differences between crab1000.py and the other mc config files?

Reply:
No I think the cfi files are compatible, and I can rename crab1000.py to be something like crab_MC.py